src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = .75f;
+    private static final int SCALE_FACTOR = 2;
+    private int size = 0;
+    private int capacity = DEFAULT_INITIAL_CAPACITY;
+    private int threshold;
```

Comment:
The `threshold` variable is declared but never used. Typically, a threshold is used to determine when to resize the hash map based on the load factor. Consider using this variable to manage resizing logic.